subprojects/android-test/ui-testing-core-app/src/androidTest/kotlin/com/avito/android/ui/test/ActivityLifecycleStatesTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.avito.android.ui.test
+
+import androidx.lifecycle.Lifecycle
+import com.avito.android.test.app.core.inHouseScreenRule
+import com.avito.android.ui.EmptyActivity
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import ru.avito.util.assertThrows
+
+class ActivityLifecycleStatesTest {
+
+    @get:Rule
+    val activityRule = inHouseScreenRule<EmptyActivity>()
+
+    @Before
+    fun before() {
+        activityRule.launchActivity(null)
+    }
+
+    @Test
+    fun finishedActivity_isInDestroyedState_succeed() {
```

Comment:
We describe another test name pattern in conventions: https://avito-tech.github.io/avito-android/contributing/Testing/#naming-conventions
`state under test (SUT) - expected behaviour - [when]`

- finishedActivity - when
- isInDestroyedState - SUT
- succeed - expected behaviour

According to this pattern, it should be `isInDestroyedState - succeed - finishedActivity`

Reply:
Yes, I didn't pay attention